### PR TITLE
[KL-500] Move backend into monorepo

### PR DESCRIPTION
**Pull request by LikoIlya**

### CODE REVIEW

File: apps/backend/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.15",
+    "@types/node": "^18.11.15",
+    "@typescript-eslint/eslint-plugin": "^5.46.1",
+    "@typescript-eslint/parser": "^5.46.1",
+    "eslint": "^8.29.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "prettier": "^2.8.1",
```
Comment (unixvb) [inline apps/backend/package.json]: seems like we don't need this deps, cuz `eslint` is configured in the root folder

File: apps/backend/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "ts-node-dev --files --quiet src/index.ts",
+    "start-prod": "node dist/index.js",
+    "backend": "node dist/index.js",
+    "backend:dev": "ts-node-dev --files --quiet src/index.ts",
+    "build": "tsc --skipLibCheck",
+    "ts": "tsc --pretty",
+    "lint": "eslint ."
```
Comment (unixvb) [inline apps/backend/package.json]: no need for this command

File: apps/backend/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  "name": "oko-wallet-backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "ts-node-dev --files --quiet src/index.ts",
+    "start-prod": "node dist/index.js",
+    "backend": "node dist/index.js",
+    "backend:dev": "ts-node-dev --files --quiet src/index.ts",
```
Comment (unixvb) [inline apps/backend/package.json:9]: why do we need `start`/`backend:dev` and `start-prod`/`backend` if they are the same?

File: apps/backend/package.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/madfish-solutions/oko-wallet-backend.git"
+  },
+  "author": "Madfish",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/madfish-solutions/oko-wallet-backend/issues"
+  },
+  "homepage": "https://github.com/madfish-solutions/oko-wallet-backend#readme",
```
Comment (unixvb) [inline apps/backend/package.json]: fix links (no more `oko-wallet-backend` repo any more)

File: libs/backend/package.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (unixvb) [inline libs/backend/package.json]: rename `libs/backend` to `libs/backend-types` seems like only types will be stored here and shared between BE and FE

File: package.json [hunk 6]
```diff
@@ -10,15 +10,21 @@
     "install:extension-e2e": "cd apps/extension-e2e && yarn",
     "install:mobile": "cd apps/mobile && yarn",
     "install:libs": "cd libs/ui && yarn",
+    "install:backend": "yarn --cwd apps/backend",
+    "install:libs-be": "yarn --cwd libs/backend",
```
Comment (unixvb) [inline package.json]: rewrite other commands using `--cwd` flag please.  
and extend `install:all` command

**Review by unixvb — APPROVED** [review]

**Review by Boltman92 — APPROVED** [review]

**Pull request merged**